crates/core/src/dht/dht_store.rs

```diff
@@ -317,15 +317,19 @@ impl DhtStore {
 
 impl GetContent for DhtStore {
     fn get_raw(&self, address: &Address) -> HcResult<Option<Content>> {
-        Ok((*self.content_storage.read().unwrap()).fetch(address)?)
+        if self.holding_list.contains(address) {
```

Comment:
I said yesterday in voice chat that I would be surprised (in a good way) if we can actually have this check here on this general level, as I was suspecting the query and fetch code to be assuming direct CAS access. 

I still haven't checked this but I see both app-spec and stress-test fail and have this error message in the logs: `Error getting entry aspects of authoring list: EntryIsPrivate`

Reply:
> I said yesterday in voice chat that I would be surprised (in a good way) if we can actually have this check here on this general level, as I was suspecting the query and fetch code to be assuming direct CAS access.
> 
> I still haven't checked this but I see both app-spec and stress-test fail and have this error message in the logs: `Error getting entry aspects of authoring list: EntryIsPrivate`

Ah interesting I think this might need a small revisit. I saw the app spec pass locally and assumed it was just flakyness on the CI side. Its really close though!
```
# tests 179
# pass  177
# fail  2
```

Reply:
and passes on a rerun... Still no luck with the stress test though

Reply:
And this was repeatable locally as well. I didn't think to run the stress tests (I assumed if app spec passed so would they. Any idea what could be causing stress tests to fail but not app spec?